keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -296,21 +584,20 @@ revert to the old behavior where rolling update will proceed one pod at a time.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-We will restore the desired behavior for StatefulSets for which the maxunavailable field wasn't deleted after 
+We will restore the desired behavior for StatefulSets for which the maxunavailable field wasn't deleted after
 the feature gate was disabled.
 
 ###### Are there any tests for feature enablement/disablement?
-yes, there are unit tests which make sure the field is correctly dropped 
+yes, there are unit tests which make sure the field is correctly dropped
```

Comment:
We usually expect integration tests, see other proposals in sig-apps territory these days. For example: https://github.com/kubernetes/enhancements/pull/3967